setup.py

```diff
@@ -68,7 +68,7 @@
     long_description_content_type="text/markdown",
     url="https://github.com/PatrikHlobil/Pandas-Bokeh",
     packages=setuptools.find_packages(),
-    install_requires=["bokeh", "pandas"],
+    install_requires=["bokeh >=1.2.0", "pandas >=0.24.2"],
```

Comment:
I think it should work with bokeh >=1.0 and pandas > 0.23. I normally develop against the most recent releases of both. I really have to try this out at some point. Maybe I could change my tox.ini file to try that ...

Could you check Bokeh 1.0 & pandas 0.23 please?

Reply:
Done, it also works with bokeh >=1.0.0 and pandas >=0.23.0